posts/data-licensing.qmd

```diff
@@ -0,0 +1,61 @@
+<!-- Data packages are often small, openly accessibly data sets that are used to showcase the functionality of a package. In these cases the data is secondary to the code, and as such many of the R packages with these "toy" data sets only state a software license.  -->
+
+Software licenses do not apply to data. One piece of data cannot be copyrighted, as facts cannot be copyrighted, but a collection of data in a database can come with rights. In Europe, UK, and Russia, databases can have rights if they are a substantial and original piece of work.[^3] This is called [the "sui generis" database right](https://en.wikipedia.org/wiki/Database_right). The USA and Brazil do not recognize this database right. Data compiled from various databases that themselves are copyrighted would need to follow the licenses of those respective databases.
+
+Data need licenses such as the Public Domain Dedication or the Creative Commons Attribution to maximize redistribution. Such [general licenses](https://r-pkgs.org/license.html#code-you-write) help minimize differences among countries that do and do not recognize database rights. Given that any copyright limitations (even attribution) mean they only apply when the database right is recognized, we recommend the most minimal option: The Public Domain Dedication. This levels the playing field for reuse and redistribution, no matter the jurisdiction.
+
+### Licensing code and data in one R package
+
+But what to do if your R package has both code and data as primary objects of (roughly) equal importance?
+
+In these cases a software license inadequately covers the data, and a data license inadequately covers the code. Dual licensing can help resolve this issue. This means there is one license for code (for example, MIT license)  and another license for the included data (for example, Public Domain Dedication).
+
+After conducting an online search, dual licensing for R packages seems rare. An interesting example of dual licensing is the [**igraphdata** package](https://github.com/igraph/igraphdata/blob/main/LICENSE), which contains several licenses: One for each dataset included in the package. Similar to igraphdata, in our own [**epiparameter** package](https://github.com/epiverse-trace/epiparameter) we dual licensed the code and data as well. We [licensed the code using the `DESCRIPTION` file](https://github.com/epiverse-trace/epiparameter/blob/4f7008063040fc4a0b436a9e4d2adb9bb7022f6c/DESCRIPTION#L21) and used [the `LICENSE` file](https://github.com/epiverse-trace/epiparameter/blob/main/LICENSE) to license the data under CC0. When including data in your R package from other sources it is important to check that the license of your package and the data is compatible[^4], or that the individual data license is clearly stated, as in igraphdata.
```

Comment:
```suggestion
After conducting an online search, dual licensing for R packages seems rare. An interesting example of dual licensing is the [**igraphdata** package](https://github.com/igraph/igraphdata/blob/main/LICENSE), which contains several licenses: One for each dataset included in the package. Similar to igraphdata, in our own [**epiparameter** package](https://github.com/epiverse-trace/epiparameter) we dual licensed the code and data as well. We [licensed the code using the `DESCRIPTION` file](https://github.com/epiverse-trace/epiparameter/blob/4f7008063040fc4a0b436a9e4d2adb9bb7022f6c/DESCRIPTION#L21) and used [the `LICENSE` file](https://github.com/epiverse-trace/epiparameter/blob/main/LICENSE) to license the data under CC0.  Concretely, we include this additional text  in `LICENSE` to clarify the dual license and that we recommend citing the original source regardless:

>  All data included in the epiparameter R package is licensed under CC0 (https://creativecommons.org/publicdomain/zero/1.0/legalcode.txt). This includes the parameter database (extdata/parameters.json) and data in the data/ folder. Please cite the individual parameter entries in the database when used.

When including data in your R package from other sources it is important to check that the license of your package and the data is compatible[^4], or that the individual data license is clearly stated, as in igraphdata.
```

@joshwlambert what do you think of this formulation based on @adamkucharski's comment?

Reply:
I think the new text is good, but I don't think the original text should be removed. I would just add it after "... under CC0." and leave the sentence "When including data in your R package ..." at the end of the paragraph. 

I think another sentence is also required to fully address Adam's point. We need to clarify that the model estimates are treated an non-copyrightable facts. This could either be added to the paragraph on L39 or this paragraph and make specific reference to {epiparameter}.

Reply:
The original text was only moved lower 😊 Here's a revised version based on your feedback. 👇 

```suggestion
After conducting an online search, dual licensing for R packages seems rare. An interesting example of dual licensing is the [**igraphdata** package](https://github.com/igraph/igraphdata/blob/main/LICENSE), which contains several licenses: One for each dataset included in the package. Similar to igraphdata, in our own [**epiparameter** package](https://github.com/epiverse-trace/epiparameter) we dual licensed the code and data as well. We [licensed the code using the `DESCRIPTION` file](https://github.com/epiverse-trace/epiparameter/blob/4f7008063040fc4a0b436a9e4d2adb9bb7022f6c/DESCRIPTION#L21) and used [the `LICENSE` file](https://github.com/epiverse-trace/epiparameter/blob/main/LICENSE) to license the data under CC0.  Concretely, we include this additional text  in `LICENSE` to clarify the dual license and that we recommend citing the original source regardless:

>  All data included in the epiparameter R package is licensed under CC0 (https://creativecommons.org/publicdomain/zero/1.0/legalcode.txt). This includes the parameter database (extdata/parameters.json) and data in the data/ folder. Please cite the individual parameter entries in the database when used.

When including data in your R package from other sources it is important to check that the license of your package and the data is compatible[^4], or that the individual data license is clearly stated, as in igraphdata. For epiparameter, we consider model estimates as facts (that is, not copyrightable).
```

@joshwlambert - could you confirm the final sentence is as you expected? If you have concrete suggestions, as you do here, it helps if you suggest them directly so there is no loss during my interpretation of your suggestion.